lambda/aws-contentgen-appsync-lambda/src/requirements.txt

```diff
@@ -0,0 +1,8 @@
+aws-lambda-powertools
```

Comment:
missing versions for dependencies

Reply:
done 3f90ec921befd1a503e473dd0a582f072c69f76b